graphics/macgui/macfontmanager.cpp

```diff
@@ -640,6 +648,41 @@ int MacFontManager::registerFontName(Common::String name, int preferredId) {
 	return id;
 }
 
+int MacFontManager::registerTTFFont(Graphics::TTFMap *ttfList) {
+	int ret = 1;
```

Comment:
Why is it `1`?

Reply:
Because in the `MacFontManager::registerFontName` function, the default ID is also set to 1.

```cpp
int MacFontManager::registerFontName(Common::String name, int preferredId) {
	// Don't register an empty font name, just return Geneva's ID.
	if (name.empty())
		return 1;
...
```